dev/PushNotifications/PushNotificationsLongRunningTask.StartupTask/PushNotificationsLongRunningTask.StartupTask.cpp

```diff
@@ -0,0 +1,20 @@
+// PushNotificationsLongRunningTask.StartupTask.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+
+int main()
+{
+    std::cout << "Hello World!\n";
```

Comment:
Remove demo code. 

Also Isn't this code supposed to active LRP?

Reply:
I though I deleted this. Okay I will delete it.